### DISCUSSION TITLE
Persona Quizzes and my Walkthroughs

### DISCUSSION THREAD

**Discussion by yayoi-chi** [jjon-saxton/yaya-world#26] [Polls]
should I make another page in the guide devoted to quizzes and exams for those who want to improve stats and leave the waklthrough as a general over view of what you should expect to be available, like noting when story scenes take over a day or what investigation steps are needed on that day and other strategies like dungeon time frames and boss stats, or should I leave quizzes and exam answer on the guide. I really want your input here. It will affect how I do other Persona guides too.

**Comment by jjon-saxton** [comment]
I don't know why, but it makes sense to me that the quiz answers are not part of the walkthrough. I've only really played part of P4 on PS2 and P5 on PS4 though.
